### ISSUE TITLE
Request for ground truth of testing data which used in RGB+TOF Depth Completion @MIPI-challenge

### ISSUE THREAD

**Issue by Lilyo** [dwHou/EMDC-PyTorch#4]
Hi @dwHou, 

Thanks for your great work! I'm curious about there is anyway to get ground truth(GT ) of testing data(not the validation_input)?

The reason I'm pursuing GT is because I want to reproduce the performance you mentioned in your paper. However, the GT depth are not available to participants in the challenge, the official said.

**Comment by dwHou** [comment]
You're welcome. The challenge is formed by two phases: validation and the final test. The GT for validation is [here](https://drive.google.com/file/d/17SWpXbnM1MLGt2AVej7ho3d-z2EoC8BF/view). The GT for the final test is not officially released, so I couldn't get it either. Fortunately, the data sources of these two phases are similar.

**Comment by Lilyo** [comment]
Hi @dwHou ,
One more quesiton, I also found the "Validset_fixed" data in official website, what's the difference between validation_input and validset_fixed? Any purpose difference?

**Comment by dwHou** [comment]
Hi, "Validset_fixed" is what I divided from the official training set, so this is a personal setting. On the other hand, the GT of validation_input was not disclosed before the end of the competition, it could not be used as a validation set at that time. 
Now that the competition is over, the official validation data should be more suitable as an offline validation set, while during the competition it can only be regarded as an online validation set as we did not have GT at that time.
